pages/intro.md

```diff
@@ -8,13 +8,644 @@ hideInToc: true
 
 <div mt-2 />
 
-- What is a Library/Framework?
-- What is React?
-- React vs other frameworks (Vue)
-- Why React?
-- From pure javascript to \*\* (the make function thing)
-- Setting up Development Environment (Vite)
+- <a href="" @click="$slidev.nav.next()">What is a Library/Framework?</a>
+- <a href="" @click="$nav.go($nav.currentPage + 0)">What is React?</a>
+- <a href="" @click="$nav.go($nav.currentPage + 0)">React vs other frameworks (Vue)</a>
+- <a href="" @click="$nav.go($nav.currentPage + 0)">Why React?</a>
+- <a href="" @click="$nav.go($nav.currentPage + 0)">From pure javascript to React</a>
+- <a href="" @click="$nav.go($nav.currentPage + 0)">Setting up Development Environment (Vite)</a>
 
 ---
 hideInToc: true
 ---
+
+# [What is a Library/Framework?]{.text-gradient}
+
+<v-clicks>
+
+- **Library**: is a collection of pre-written code that developers can call upon to perform specific tasks. Libraries offer various functionalities, but you are in control of how and when to use them. {React} is often considered a library because it focuses on rendering the UI. It doesn't provide everything needed to build an application, but it does provide a way to build user interfaces. {Vue}, {Angular}, and {Svelte} are other examples of libraries. {jQuery} is another example of a library that provides functionalities like DOM manipulation, event handling, and AJAX calls.
+
+- **Framework**: A framework is a more opinionated structure or skeleton for building applications. It typically provides everything needed (like a routing system, data management, etc.) and controls the flow of the application. Although {Nextjs} uses {React}, it is considered a framework because it provides a predefined structure for building applications, including file-based routing, server-side rendering, and static site generation. {Nuxt} is another example of a framework that uses {Vue}.
+
+Javascript libraries and frameworks are tools that help developers build applications faster and more efficiently. They provide a set of functionalities that can be reused across different projects, saving time and effort.
```

Comment:
Super impressive style to include external links in the `vite.config.ts` and easily access them here with just curly braces ({React}). My comment is on the default styling which leaves a `left-margin` that causes the unintended spaces disfiguring the look of the note. I believe this my be in the `style.css` file where the default style for links are set, if yes, you can consider checking it out. If this is nothing to worry about in your opinion, we can ignore it.

Reply:
I resolved the top padding on `markdow-magic-link` class in the stylesheet.